package.json

```diff
@@ -9,6 +9,7 @@
   "main": "index.js",
   "scripts": {
     "start": "node index.js",
+    "start:cloudflare": "npm run build:fast && npm install wrangler && npx wrangler pages dev build/",
```

Comment:
Who should be using this script vs `deploy-docs` used by Flowzone? Is this one for local development or manual deployment or something?

Reply:
Indeed, this command is for local development 
For deployment we are using deploy-docs.